File: TestGen_java/Employee.java
public class Employee {
    private String name;
    private double salary;

    public Employee(String name, double salary) {
        this.name = name;
        this.salary = salary;
    }

    public void work() {
        System.out.println(name + " is working on something.");
    }

    public String getDetails() {
        return "Employee: " + name + ", Salary: $" + salary;
    }

    // Getters and Setters (optional)
    public String getName() {
        return name;
    }

    public double getSalary() {
        return salary;
    }
}

File: TestGen_python/employee.py
class Employee:
    def __init__(self, name, salary):
        self.name = name
        self.salary = salary

    def work(self):
        print(f"{self.name} is working.")

    def get_details(self):
        return f"Employee: {self.name}, Salary: ${self.salary}"
